spec/cassandra/protocol/coder_spec.rb

```diff
@@ -37,11 +37,20 @@ module Protocol
       end
 
       describe('RUBY-128') do
+        let(:permitted_classes) { [Cassandra::Protocol::CqlByteBuffer, Cassandra::Types::Map, Cassandra::Types::Simple, Symbol] }
```

Comment:
total nit pick, feel free to ignore, but I'd make this a variable inside the `if Psych` branch since it's only used there. Generally you reserve let blocks for setup or assertions across multiple specs.

Reply:
I'm going to ignore, but I aspire to a higher level of rspecing